release_notes/unreleased.md

```diff
@@ -1 +1,27 @@
-**Unreleased**
+<b>Trend Micro Vision One Release Notes - Published by Trend Micro July 01, 2022</b>
```

Comment:
The formatting seems to be incorrect. Can we change it?

Reply:
Done